src/write.c

```diff
@@ -1831,11 +1831,24 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
     }
 
     // -----------------------------------------------------------------------
-    // Map quality and qualityAlpha to quantizer and quantizerAlpha
+    // Map quality settings to quantizer values.
     encoder->data->quantizer = avifQualityToQuantizer(encoder->quality, encoder->minQuantizer, encoder->maxQuantizer);
-    encoder->data->quantizerAlpha = avifQualityToQuantizer(encoder->qualityAlpha, encoder->minQuantizerAlpha, encoder->maxQuantizerAlpha);
-    encoder->data->quantizerGainMap =
-        avifQualityToQuantizer(encoder->qualityGainMap, AVIF_QUANTIZER_BEST_QUALITY, AVIF_QUANTIZER_WORST_QUALITY);
+    // If alpha quality, and min and max quantizer have their default values, default to the same quality as color.
```

Comment:
Nit: min and max quantizer => alpha min and max quantizers

Alternatively, we can say "If alpha quality, min quantizer, and max quantizer ..."